src/test/resources/com/cognifide/gradle/aem/test/hosts/hosts

```diff
@@ -0,0 +1,9 @@
+##
```

Comment:
generally vagrant hostmanager plugin and other stuff like this (e.g https://github.com/scottlerch/HostsFileEditor ) are wrapping theirs entries inside commented section

```
# gap-managed-hosts-start
...
#gap-managed-hosts-start
```

sth like this